src/cryptography/hazmat/bindings/openssl/binding.py

```diff
@@ -12,6 +13,42 @@
 from cryptography.hazmat.bindings.openssl._conditional import CONDITIONAL_NAMES
 
 
+_OpenSSLError = collections.namedtuple("_OpenSSLError",
+                                       ["code", "lib", "func", "reason"])
+
+
+class UnhandledOpenSSLError(Exception):
```

Comment:
What in the hell. I swear this diff looks different now (but obviously I'm wrong). Ignore me.